dist/dev/index.html

```diff
@@ -1,26 +1,47 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta charset="utf-8">
-  <link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">
-  <link href="/css/main-index.css" rel="stylesheet" type="text/css"/>
-  <link href="/css/main-fonts.css" rel="stylesheet" type="text/css"/>
-  <link href="/css/slick.min.css" rel="stylesheet" type="text/css"/>
-  <link href="/css/slick-theme.min.css" rel="stylesheet" type="text/css"/>
-  <meta property="og:image" content="https://s3.amazonaws.com/stuyspec-images/the_logo.png"/>
-  <meta property="og:title" content="The Stuyvesant Spectator" />
-  <meta property="og:url" content="https://stuyspec.com" />
-  <meta property="og:site_name" content="The Spectator" />
-  <meta property="og:type" content="website" />
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta name="author" content="The Spectator Web Department" />
+    <meta name="description" content="The Pulse of the Student Body" />
+    <meta name="keywords" content="The Spectator, journalism, Stuyvesant High School, student body" />
+
+    <meta property="og:image" content="https://s3.amazonaws.com/stuyspec-images/the_logo.png"/>
+    <meta property="og:title" content="The Stuyvesant Spectator" />
+    <meta property="og:url" content="https://stuyspec.com" />
+    <meta property="og:site_name" content="The Spectator" />
+    <meta property="og:type" content="website" />
+
+    <title>The Stuyvesant Spectator - The Pulse of the Stuyvesant Student Body</title>
+
+    <link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">

```

Comment:
We should try to combine these stylesheets. That way we can try to avoid the blast of non formatted content

Reply:
Combined the slick ones into main-index.css, but combining bootstrap breaks the site.